extensions/amp-access/0.1/amp-access-server.js

```diff
@@ -180,6 +193,39 @@ export class AccessServerAdapter {
     return this.clientAdapter_.pingback();
   }
 
+  /** @override */
+  encodeReaderId(readerId) {
+    return this.getSubtleWithKey_().then(key => {
+      return this.subtle_.encrypt(key, readerId).then(s => {
```

Comment:
Let's extract after this PR. I'd like to get the first experimental version out to confirm few implementation notes first before we affect everyone else who is using `amp-analytics`.